### PR TITLE
Add docs; Add Account entity; Process orders instead of domains; Update tests

### PR DESCRIPTION
**Pull request by erkangz**

### CODE REVIEW

File: test/polly.ts [hunk 1]
```diff
@@ -9,9 +9,9 @@ Polly.register(FSPersister);
 
 export { Polly } from '@pollyjs/core';
 
-class TrendMicroFSPersister extends FSPersister {
+class DigiCertFSPersister extends FSPersister {
```
Comment (aiwilliams) [inline test/polly.ts:12]: The `integration-sdk` include all the code necessary for doing this now, https://github.com/JupiterOne/integration-sdk/blob/master/src/testing/recording.ts#L51, also documented here if that helps: https://github.com/JupiterOne/integration-sdk/blob/master/docs/testing.md#setuprecording

**Comment by aiwilliams** [comment]
Hmm, looks like something is missing in the tests that do recording. The build is failing because the tests are making actual calls to the digicert APIs, which shouldn't be happening in CI/CD builds. This was happening to someone else this week, but I fail to recall who, and I don't know what they did to resolve it. I would recommend using the `setupRecording` function from the SDK, and review how tests such as those in `graph-trend-micro` are written, assuming it's not actually making requests 😬

File: test/polly.ts (hunk 1 above)
Comment (erkangz) [reply test/polly.ts]: thanks. updated.

File: src/steps/fetch-domains/__tests__/index.test.ts [hunk 2]
```diff
@@ -1,33 +1,57 @@
...
-      name: 'test.com',
+      _class: ['Certificate'],
+      domainId: 1587137,
       active: true,
-      displayName: 'test.com',
-      id: '1564602',
-      createdOn: 1587412402000,
-      expiresOn: 1658620800000,
+      displayName: 'erkang.com',
+      organization: 'Erkang, LLC',
```
Comment (aiwilliams) [inline src/steps/fetch-domains/__tests__/index.test.ts:51]: Starting your empire, eh? 😁

Comment (erkangz) [reply src/steps/fetch-domains/__tests__/index.test.ts]: 👍

**Pull request merged**